src/filter_parameters.rs

```diff
@@ -19,8 +25,85 @@ pub struct FilterParameters {
     pub drive: ParameterF32,
     pub mode: ParameterUsize,
     pub slope: ParameterUsize,
-    pub filter_type: AtomicUsize,
+    pub filter_type: ParameterUsize,
+
+    pub reserved: ParameterUsize,
+}
+
+#[repr(i32)]
+#[derive(FromPrimitive, Eq, PartialEq, Debug, EnumIter)]
+pub enum FilterParameterNr {
+    Cutoff,
+    Res,
+    Drive,
+
+    FilterType = 32,
```

Comment:
Cool that this is possible, but it's probably better not to do it since it confuses hosts